orgs/altinn-orgs.json

```diff
@@ -29,6 +29,18 @@
             "orgnr": "999999991",
             "environments": []
         },
+        "bits": {
+            "name": {
+                "en": "BITS AS",
+                "nb": "BITS AS",
+                "nn": "BITS AS"
+            },
+            "logo": null,
```

Comment:
Unsure about the consequence of this - can we add a logo to the orgs folder under orgs/bits